.project

```diff
@@ -14,4 +14,15 @@
 	<natures>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
Same question as in the other review. Thoughts on putting this in gitignore? Thanks

Reply:
Yeah, my issue is committing project files in every commit. If we can find a way around that (like committing in a separate PR), then I'm good. Thanks